lib/origen_testers/igxl_based_tester/base.rb

```diff
@@ -717,7 +717,11 @@ def pattern_footer(options = {})
         $tester.align_to_last
         # cycle(:microcode => "#{$dut.end_of_pattern_label}:") if $dut.end_of_pattern_label
         if options[:end_in_ka]
+          if $tester.j750?
```

Comment:
Updates have been pushed and the PR is now updated. Thanks for reviewing!

I am hoping we can have a merge/release in the next couple of days.